src/lib/dlt_user.c

```diff
@@ -2042,7 +2058,7 @@ static DltReturnValue dlt_user_log_write_generic_attr(DltContextData *log, const
     if (log == NULL)
         return DLT_RETURN_WRONG_PARAMETER;
 
-    if (!DLT_USER_INITALIZED) {
+    if (!DLT_USER_INITALIZED_NOT_FREEING) {
         dlt_vlog(LOG_WARNING, "%s dlt_user_init_state != INIT_DONE\n", __FUNCTION__);
```

Comment:
Minor remark: 
this can now fail due to many different reasons. it would be good to log the state of 
dlt_user_freeing and dlt_user_init_state as well.

Reply:
similar messages is used in other places. if you like the message I'll change in these places as well.

Reply:
I applied the new message to all applicable lines.